dht/range_streamer.hh

```diff
@@ -78,7 +79,7 @@ public:
 
     range_streamer(distributed<replica::database>& db, sharded<streaming::stream_manager>& sm, const token_metadata_ptr tmptr, abort_source& abort_source, std::unordered_set<token> tokens,
             inet_address address, locator::endpoint_dc_rack dr, sstring description, streaming::stream_reason reason,
-            std::vector<sstring> tables = {})
+            std::vector<sstring> tables = {}, service::frozen_topology_guard topo_guard = {})
```

Comment:
Should be non-optional to avoid untracked sessions by accident.

Reply:
Ok